jvm/src/main/scala/kiama/util/Server.scala

```diff
@@ -122,6 +125,14 @@ trait Server[N, C <: Config, M <: Message] extends Compiler[C, M] with LanguageS
     if (config.debug()) {
       import java.net.InetSocketAddress
       import java.nio.channels.{ AsynchronousServerSocketChannel, Channels }
+      import java.util.logging.{Logger, Level, ConsoleHandler}
+
+      // Set up LSP logging
+      val log = Logger.getLogger("LSP")
+      log.setLevel(Level.ALL)
+      val handler = new ConsoleHandler()
+      handler.setLevel(Level.ALL)
+      log.addHandler(handler)
```

Comment:
This enables logging of LSP commands when in `run --server --debug` mode in sbt.

This looks like:

```
[Trace - 03:43:22 PM] Received notification 'notebookDocument/didChange'
Params: {
  "notebookDocument": {
    "version": 62,
    "uri": "file:///Users/jonathan/Desktop/github/effekt-vscode-notebook/examples/note.effektbook"
  },
  "change": {
    "cells": {
      "textContent": [
        {
          "document": {
            "version": 3,
            "uri": "vscode-notebook-cell:/Users/jonathan/Desktop/github/effekt-vscode-notebook/examples/note.effektbook#X11sZmlsZQ%3D%3D"
          },
          "changes": [
            {
              "range": {
                "start": {
                  "line": 0,
                  "character": 1
                },
                "end": {
                  "line": 0,
                  "character": 1
                }
              },
              "rangeLength": 0,
              "text": "2"
            }
          ]
        }
      ]
    }
  }
}
```